packages/package-json-lint/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Added `type` to allow users to flag project as `application`, `module` or `devModule`.
```

Comment:
Should we name this flag `projectType` to be more explicit?

Reply:
@benbcai, good call yeah let's do that.

Reply:
Updated here https://github.com/cerner/terra-toolkit/pull/700/commits/8ad7e5e9e43dce7f25ad36999a0395157ccbb5e1